src/Symfony/Component/RateLimiter/RateLimit.php

```diff
@@ -67,6 +67,6 @@ public function getLimit(): int
 
     public function wait(): void
     {
-        sleep(($this->retryAfter->getTimestamp() - time()) * 1e6);
+        usleep(($this->retryAfter->getTimestamp() - time()) * 1e6);
```

Comment:
Let's make this simpler: 

```suggestion
        sleep($this->retryAfter->getTimestamp() - time());
```

Alternatively, we decide that we want to support microseconds and use `usleep` and `microtime`

Reply:
indeed, time() has `1 sec` precision.

Thanks